ansible/roles/satellite-configuration/tasks/01_manifest.yml

```diff
@@ -0,0 +1,33 @@
+---
+
+- name: fetching list of existing subscriptions
+  shell: hammer --output cve subscription list --organization "{{ORG}}"
```

Comment:
`s/shell/command/` (you're not using any shell feature)

Reply:
replaced shell with command module